app/javascript/packs/pw-strength.js

```diff
@@ -26,13 +26,21 @@ function getStrength(z) {
   return z && z.password.length ? scale[z.score] : fallback;
 }
 
-export function getFeedback(z) {
-  if (!z || !z.password || z.score > 2) {
+export function getFeedback(z, { minimumLength }) {
+  if (!z || !z.password) {
     return '&nbsp;';
   }
 
   const { warning, suggestions } = z.feedback;
 
+  if (!warning && !suggestions.length) {
+    if (z.password.length < minimumLength) {
+      return t('errors.attributes.password.too_short.other', { count: minimumLength });
```

Comment:
We might have to either update this message text or use a separate string, since the Figma design has this as different from the error message that already exists.

Figma design: "Password must be at least 12 characters long"
This branch: "This password is too short (minimum is 12 characters)"